doc/content/devices/atecc608a/transfer.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Transfer to new Join Server"
+description: ""
+weight: 2
+distributions: ["Enterprise", "Cloud"]
+---
+
+ATECC608A/B secure elements are provisioned to The Things Join Server. This Join Server supports activation with all LoRaWAN Backend Interfaces compliant Network Servers. You don't need to be a user of {{% tts %}} to activate devices with a secure element via The Things Join Server.
+
+<!--more-->
+
+If you don't want to use The Things Join Server, you can transfer to a new Join Server. There are two options for doing so, which are both explained in this section.
+
+## New `JoinEUI`
+
+Regardless of how the root keys are transferred, a new Join Server requires a new `JoinEUI` (or `AppEUI`) for the end devices. This is because the `JoinEUI` with which the secure elements are provisioned points exclusively to The Things Join Server.
```

Comment:
```suggestion
Regardless of how the root keys are transferred, a new Join Server requires a new `JoinEUI` (or `AppEUI`) for the end devices. This is because the `JoinEUI`, which the secure elements are provisioned with, points exclusively to The Things Join Server.
```